### PR TITLE
refact: replace react-json-view with react-18-compat lib

### PR DESCRIPTION
**Pull request by davidlougheed**
also moves react18-json-view usage into fewer locations

### CODE REVIEW

File: src/components/explorer/IndividualBiosamples.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ import {
 } from "@/propTypes";
 import { useDeduplicatedIndividualBiosamples } from "./utils";
 
-import JsonView from "./JsonView";
+import JsonView from "../JsonView";
```
Comment (v-rocheleau) [inline src/components/explorer/IndividualBiosamples.js:16]: Use `@` import alias

File: src/components/explorer/IndividualExperiments.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ import { guessFileType } from "@/utils/files";
 import { useDeduplicatedIndividualBiosamples } from "./utils";
 import { VIEWABLE_FILE_EXTENSIONS } from "../display/FileDisplay";
 
-import JsonView from "./JsonView";
+import JsonView from "../JsonView";
```
Comment (v-rocheleau) [inline src/components/explorer/IndividualExperiments.js:17]: Use `@` import alias

File: src/components/explorer/IndividualVariants.js [hunk 3]
```diff
@@ -11,7 +11,7 @@ import "./explorer.css";
 import { ExplorerIndividualContext } from "./contexts/individual";
 import { explorerIndividualUrl } from "./utils";
 
-import JsonView from "./JsonView";
+import JsonView from "../JsonView";
```
Comment (v-rocheleau) [inline src/components/explorer/IndividualVariants.js:14]: Use `@` import alias

File: src/components/manager/projects/ProjectJsonSchema.js [hunk 4]
```diff
@@ -76,13 +76,7 @@ const ProjectJsonSchema = ({ projectSchema }) => {
...
 
                 <Descriptions.Item label="JSON Schema">
-                    <ReactJson
-                        src={projectSchema.json_schema ?? {}}
-                        collapsed={true}
-                        displayDataTypes={false}
-                        enableClipboard={false}
-                        name={null}
-                    />
+                    <JsonView src={projectSchema.json_schema ?? {}} collapsed={true} />
```
Comment (v-rocheleau) [inline src/components/manager/projects/ProjectJsonSchema.js:79]: `JsonView`'s `collapsed` prop is true by default.

**Review by v-rocheleau — CHANGES_REQUESTED** [review]

File: src/components/manager/projects/ProjectJsonSchema.js (hunk 4 above)
Comment (davidlougheed) [reply src/components/manager/projects/ProjectJsonSchema.js:79]: it's 1 by default, which means 1 level of nesting is expanded (the top level)

**Pull request merged**